src/Common/src/Interop/OSX/libc/Interop.sync.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I believe ```sync``` is POSIX compliant.  Consider putting it in the Unix rather than OSX directory.